### DISCUSSION TITLE
Where to put executables that shall be accessible by AzuraCast’s Liquidsoap?

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6940] [Q&A] [answered]
I’m trying for an "on-the-fly" cue point, cross duration and `liq_amplify` (i.e., ReplayGain) calculator that shall be accessible in AzuraCast’s Liquidsoap configuration. It will require both _Python3_ and _ffmpeg_, which we both have within the Docker.

@BusterNeece Would it feel right to place that into `/var/azuracast/bin` on the host and overlay it into `/usr/local/bin` within the Docker, using the `docker-compose.override.yaml`?

And do you deem roughly 2–3 s computing time per average 3–5 minute song acceptable? (My longest song, a 79:12 486 MB FLAC file, takes ~24 s to calculate.)

**Comment by BusterNeece** [comment, marked as answer]
Just to answer this question, I'd recommend mounting any custom executables into `/usr/local/bin` so they're still in the PATH but not interfering with a ton of other binaries. The AzuraCast binaries also live in there, so make sure not to remount the *whole* folder, just the specific executables you want to add.
